### DISCUSSION TITLE
RTSP stream traffic

### DISCUSSION THREAD

**Discussion by kobius77** [mcguirepr89/BirdNET-Pi#894] [General]
having BirdNET-Pi running in a VM consuming an RTSP-stream from a camera for some days now i found the strange effect of a considerable rise of incoming network traffic at night.

![image](https://user-images.githubusercontent.com/3143680/236701389-a1096679-90a3-479b-8328-565488a0a8d8.png)

my wild guess is that this is due to the video of the camera being more noisy at night - is this plausible? does anyone know if there are ways to reencode such stream to audio only? i actually am connected to that stream with another VM on that host, and another device on the network, so it might make more sense to reencode than stuffing the wifi.

besides, i'm seeing a strange rise in cpu-usage since yesterday evening but that probably isn't related to the stream. will have to keep an eye on it, aware that running in a VM isn't what this is made for.
![image](https://user-images.githubusercontent.com/3143680/236701720-c2d57115-581d-4c7d-a31c-90b80492999e.png)

also did have a strange period of zero detections this morning till 9am which i didn't bother investigating yet - or find out how to do it.

**Comment by ehpersonal38** [comment]
> my wild guess is that this is due to the video of the camera being more noisy at night - is this plausible? 

Maybe, I'm not really sure. Is the audio louder at night by chance? Here it can actually be quieter during the day than at night when the crickets come out. But I doubt that would account for the quadruple in bandwidth. It's a mystery for me!

> does anyone know if there are ways to reencode such stream to audio only? i actually am connected to that stream with another VM on that host, and another device on the network, so it might make more sense to reencode than stuffing the wifi.

BirdNET-Pi basically already hosts its own audio-only re-stream of your RTSP stream, located at `http://{your birdnet-pi web interface link}/stream`. Would that work?

**Reply by kobius77** [reply]
> Maybe, I'm not really sure. Is the audio louder at night by chance? Here it can actually be quieter during the day than at night when the crickets come out. But I doubt that would account for the quadruple in bandwidth. It's a mystery for me!

no, it's definitly more silent at night, too cold for crickets yet. and less road noise. but the video is just a dark mass of noise which probably is harder to compress than what is recorded by daylight. 



> BirdNET-Pi basically already hosts its own audio-only re-stream of your RTSP stream, located at `http://{your birdnet-pi web interface link}/stream`. Would that work?

ah - right - will have a look at how it's done there.
